JuliaGPU/session2/Adharsh Kamath/q3.jl

```diff
@@ -0,0 +1,49 @@
+using Test
+Array = rand(2^20)
+
+function squareroot(number)
+    startNum = 0
+    endNum = number
+    answer = 0
+    while startNum <= endNum
+        middle = div((startNum + endNum),2)
+        if middle*middle == number
+            answer = middle 
+            break
+        end
+        if middle*middle < number 
+            startNum = middle + 1 
+            answer = middle
+        else
+            endNum = middle - 1 
+        end
+    end
+    
+    increment = 0.1; 
+    for i = 0:9 
+        while answer*answer <= number 
+            answer += increment
+        end
+        answer = answer-increment 
+        increment = increment/10     
+    end
+    return answer
+    end
+
+array_sqrt(Array) = begin
+       emptyArray = []
+       for each in Array
```

Comment:
It's almost correct. The question asks for a mutating function.  
Also, it is a bit messy. Could you define array_sqrt in such that the definition just takes one line?